.metadata.toml

```diff
@@ -1359,6 +1359,27 @@ examples = [{ name = "X-Powered-By", value = "Vector"}]
 null = false
 description = "A custom parameter to be added to each Elasticsearch request."
 
+# ------------------------------------------------------------------------------
+# sinks.file
+# ------------------------------------------------------------------------------
+[sinks.file]
```

Comment:
@binarylogic Seems like we are missing `close_timeout_secs` and `encoding` options. I can add them tomorrow if that is the case.